bench/tx-generator/src/Cardano/Benchmarking/Script/Action.hs

```diff
@@ -47,8 +46,8 @@ action a = case a of
   AddFund era wallet txIn lovelace keyName -> addFund era wallet txIn lovelace keyName
   Delay t -> delay t
   Submit era submitMode txParams generator -> submitAction era submitMode generator txParams
-  WaitBenchmark thread -> waitBenchmark thread
-  CancelBenchmark thread -> cancelBenchmark thread
+  WaitBenchmark _thread -> waitBenchmark
+  CancelBenchmark _thread -> cancelBenchmark
```

Comment:
Why not remove those obsolete args?
to/fromJSON instances and `bench/tx-generator/test/script.json` would probably need minor adjustments then.

Reply:
I also folded this back into the commit touching these lines in a rebase. Somehow I ended up not pushing this until Monday morning in New York despite its having been completed on Friday.